src/js/queries/getPageRevisionQuery.js

```diff
@@ -0,0 +1,125 @@
+import servicePageFragment from './servicePageFragment';
+import departmentPageFragment from './departmentPageFragment';
+import eventPageFragment from './eventPageFragment';
+import formContainerFragment from './formContainerFragment';
+import guidePageFragment from './guidePageFragment';
+import informationPageFragment from './informationPageFragment';
+import locationPageFragment from './locationPageFragment';
+import officialDocumentPageFragment from './officialDocumentPageFragment';
+import topicCollectionFragment from './topicCollectionFragment';
+import topicFragment from './topicFragment';
+
+// todo in the distant future:
+// * figure out duplicate fragment issues
+// * get this working
```

Comment:
is it not working because the fragments need to be updated?
or does it not like this format?
i want to do something similar for allPagesQuery

Reply:
an example of why it's broken:
multiple page types have contacts, we use contact fragments in multiple queries, when they're all assembled here we end up having included the contact fragment multiple times, causing errors.

Reply:
That's interesting that we're not allowed to use a fragment more than once in a query. We could make hacky fragments like I did in gql in Joplin. But hopefully there's a non-hacky solution.